index.html

```diff
@@ -2252,7 +2252,7 @@ <h2 id="docconformance">
             <td>
               <p>
                 If a `label` element is implicitly or explicitly associated with a 
-                <a href="data-html/https://html.spec.whatwg.org/multipage/forms.html#category-label">labelable element</a> then
+                <a href="https://html.spec.whatwg.org/multipage/forms.html#category-label">labelable element</a> then
```

Comment:
wow, i must have been distracted when i wrote that before.

this is what it should have been.
```suggestion
                <a data-cite="html/forms.html#category-label">labelable element</a> then
```